### PR TITLE
Update some docs to Windows 11 and adjust terms to style guide

### PR DESCRIPTION
**Pull request by noraa-junker**
Asjusts docs to style guide (Term: "and later")

Updated all left docs from the "apps" folder.

> This PR is part of my effort to bring all Microsoft Docs to Windows 11. You can see my overall progress in [this gist](https://gist.github.com/Aaron-Junker/ad8f6234f1a991e46e234b25dc0ce0dc)

### CODE REVIEW

**Comment by jborsecnik** [comment]
@mcleanbyron - Would you review and approve this submission? To indicate your approval, add a sign off comment.

This submission is mostly modest style changes, but two topics need technical review.
These articles don't have listed authors:
- **net-projection-from-cppwinrt-component.md**
- **user-interface.md**

If you're not able to approve these changes, can you suggest an alternate reviewer / approver?
Thank you

File: hub/apps/design/shell/tiles-and-notifications/send-local-toast-desktop-cpp-wrl.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ In the top **Configuration** menu, select **All Configurations** so that the fol
 
 Under **Linker -> Input**, add `runtimeobject.lib` to the **Additional Dependencies**.
 
-Then under **General**, make sure that the **Windows SDK Version** is set to something 10.0 or higher (not Windows 8.1).
+Then under **General**, make sure that the **Windows SDK Version** is set to something 10.0 or later (not Windows 8.1).
```
Comment (Jay-o-Way) [inline hub/apps/design/shell/tiles-and-notifications/send-local-toast-desktop-cpp-wrl.md]: Something? What?

File: windows-apps-src/gaming/package-your-windows-store-directx-game.md [hunk 2]
```diff
@@ -33,7 +33,7 @@ Content candidates for game resource packs include:
 
 -   International locale specific assets (localized text, audio, or images)
 -   High resolution assets for different device scaling factors (1.0x, 1.4x, and 1.8x)
--   High definition assets for higher DirectX feature levels (9, 10, and 11)
```
Comment (Jay-o-Way) [inline windows-apps-src/gaming/package-your-windows-store-directx-game.md:36]: Not sure, don't have much context, but i think levels _are_ higher/lower?

File: windows-apps-src/graphics-concepts/texture-addressing-modes.md [hunk 3]
```diff
@@ -23,7 +23,7 @@ You control what Direct3D does with texture coordinates that are outside the \[0
 
 Direct3D enables applications to perform texture wrapping. See [Texture wrapping](texture-wrapping.md).
 
-Enabling texture wrapping effectively makes texture coordinates outside the \[0.0, 1.0\] range invalid, and the behavior for rasterizing such delinquent texture coordinates is undefined in this case. When texture wrapping is enabled, texture addressing modes are not used. Take care that your application does not specify texture coordinates lower than 0.0 or higher than 1.0 when texture wrapping is enabled.
+Enabling texture wrapping effectively makes texture coordinates outside the \[0.0, 1.0\] range invalid, and the behavior for rasterizing such delinquent texture coordinates is undefined in this case. When texture wrapping is enabled, texture addressing modes are not used. Take care that your application does not specify texture coordinates lower than 0.0 or later than 1.0 when texture wrapping is enabled.
```
Comment (Jay-o-Way) [inline windows-apps-src/graphics-concepts/texture-addressing-modes.md]: "lower" and "higher" do belong together here.

File: windows-apps-src/publish/age-ratings.md [hunk 4]
```diff
@@ -48,6 +48,6 @@ Previously, developers specified the age rating for their app rather than comple
...
+| 12+ (Suitable for ages 12 and older) | Apps with this age rating can contain increased nudity of a non-sexual nature, slightly graphic violence towards non-realistic characters, or non-graphic violence towards realistic human or animal characters. This age rating might also include profanity, but not of a sexual nature. Also, apps with this age rating or later may allow for uncontrolled: (i) access to online social networks, or (ii) sharing of personal info with third parties, including other gamers or online acquaintances. (For such activity to be considered controlled, your app must include parental control features that require parental permission to use such sharing features, and you must identify those and explain their functionality in the [Notes for certification](notes-for-certification.md).) |
```
Comment (Jay-o-Way) [inline windows-apps-src/publish/age-ratings.md]: Don't think this should be changed

**Comment by mattwojo** [comment]
@hickeys - Looks like these updates are primarily to design docs, so removed McLean and assigned to you. Please feel welcome to assign others as needed or just review and merge if everything looks good.

**Review by GrantMeStrength — APPROVED** [review]

**Comment by GrantMeStrength** [comment]
Karl - before I take this fix, there are three conflicts that mention WinUI 3. I don't want to accidentally overwrite your change.

**Comment by jborsecnik** [comment]
@GrantMeStrength, I held off on this to make sure it's not merged prematurely. Please add `#sign-off` when it's ok to merge.

#hold-off

**Comment by Karl-Bridge-Microsoft** [comment]
@Aaron-Junker - please sync with main and update your PR. There are old files causing conflicts. Thanks.

**Review by Karl-Bridge-Microsoft — APPROVED** [review]

**Comment by Karl-Bridge-Microsoft** [comment]
Thanks @Aaron-Junker. Due to the unfortunate staleness of this PR, I duplicated your updates in a private PR and merged.

Closing this PR. Please open a new one if anything was missed.

**Comment by noraa-junker** [comment]
> Thanks @Aaron-Junker. Due to the unfortunate staleness of this PR, I duplicated your updates in a private PR and merged.
> 
> Closing this PR. Please open a new one if anything was missed.

Thank you @Karl-Bridge-Microsoft I'm sorry I completely forgot this. I will double check if all changes got merged.

Thanks!

**Pull request closed without merging**